bcauth/jinja2/account/email_confirmed.html

```diff
@@ -4,7 +4,6 @@
 {% block body_title %}{{ _("Confirm E-mail Address") }}{% endblock %}
 
 {% block content %}
-{% user_display confirmation.email_address.user as user_display %}
```

Comment:
Oh, whoops--I missed the statement about csrf token issues. That's a bug that we should fix with this PR. I'll add it to the todo list.